Update toggle docs with new label guidelines (#621)
vtex/styleguide: react/components/Toggle/README.md

#### A Toggle is a control for turning an option ON or OFF. Its effect is usually automatically applied, which is one of the main differences compared to checkboxes, that need a submit.

### 👍 Dos

> - The label should reflect the component state, like "On/Off", or "Enabled/Disabled", it shouldn't in imperative form (like "Enable").

Reviewer: ```suggestion
- The label should reflect the component state, like "On/Off", or "Enabled/Disabled", it shouldn't use an imperative tone (like "Enable").
```
